tests/unit-fail-compilation/dup_params1.ref

```diff
@@ -1,7 +1,7 @@
 File: dup_params1.c
 Line: 1
-Error: CV-FD1
-Description: duplicate parameters in function definition.
+Error: CV-TDL6
```

Comment:
Okay. Then yeah, make sure you check the example to ensure that it is still triggering the right error and update it if necessary.